### ISSUE TITLE
Improve/expand definition of members of standing

### ISSUE THREAD

**Issue by jdumars** [kubernetes/steering#48]
- Improve/expand definition of members of standing
- Determine how this is maintained, especially during elections that require it

**Comment by timothysc** [comment]
Closing this issue in favor of the meta issue to lump the items together.

**Comment by bgrant0607** [comment]
Timeline: https://github.com/kubernetes/community/issues/3915

Process: https://github.com/kubernetes/community/tree/master/events/elections

Voting eligibility:
https://github.com/kubernetes/steering/blob/master/elections.md#eligibility-for-voting

The bar is currently 50 contributions, which is a little over 800 voters, which is only a little more than last year:
https://k8s.devstats.cncf.io/d/13/developer-activity-counts-by-repository-group?orgId=1&var-period_name=Last%20year&var-metric=contributions&var-repogroup_name=All&var-country_name=All

Per @parispittman and @castrojo, there weren't many exceptions last year, and they were pretty clear, so maybe the exception process is ok also.

**Comment by spiffxp** [comment]
OK, I can live with 50+ devstats contributions, and a generous exception process.

Drilling through other options for member of standing criteria (using data from 2019-07-31 which is where we capped it last year):
- any org membership OR 50+ devstats: 1231
- any org membership: 1022
- kubernetes org membership: 934
- 50+ devstats: 768
- any org membership AND 50+ devstats: 559

For historical reference:
- 2018: we chose 50+ devstats, ended up with 692 eligible, 307 voted (44% turnout)
- 2017: not sure what the criteria was, ended up with 392 eligible, 309 voted (79% turnout)

I found adding org membership as a requirement excluded a lot of drive-by contributions, but also highlighted a few active contributors who aren't org members.  I guess I'd rather err on getting org membership correct before adding that as a requirement.

The linked sheet has (kubernetes, any org, 50+ devstats, devstats) and can be used to filter to explore if choosing something more restrictive filters out meaningful contributors by these arbitrary metrics. Not included are the ~3000 contributors with less than 50 devstats https://docs.google.com/spreadsheets/d/1TKeKucfSwhP8NS4qAskRIN_VhD-dK_DhqwVv_I9xsU4/edit#gid=1277515370
